bom/pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>org.apache.cxf</groupId>
         <artifactId>cxf</artifactId>
-        <version>4.0.1-SNAPSHOT</version>
```

Comment:
@karlvr I've bumped the version to 4.1.0 to simplify branch switches, it expanded the changeset quite a bit, apologies for that